lib/ansible/modules/network/nxos/nxos_aaa_server_host.py

```diff
@@ -106,9 +106,9 @@
         tacacs_port: 89
         host_timeout: 10
         address: 5.6.7.8
-        host:  inventory_hostname }}
-        username:  un }}
-        password:  pwd }}
+        host: "{{ inventory_hostname }}"
+        username: "{{ un }}"
+        password: "{{ pwd }}"
```

Comment:
the top level spec actually needs to removed from example as we have deprecated them.

Reply:
@trishnaguha - can you add more detail to your request? This came in as a docsite_pr so the owner may not follow what you are suggesting (and I'm not sure what you want changed either :-)  Thanks!

Reply:
okay read the page a bit more - are you saying host, username, and password should be removed from the examples entirely?

Reply:
@samccann Yes that's correct. It should be removed from the task example entirely. As we have deprecated top level spec we don't want to encourage user to use it.